### PR TITLE
#166789823 Setup react app

### PR DESCRIPTION
**Pull request by adafia**
#### What does this PR do?
This pull request sets up a simple react application with webpack and babel configurations with Airbnb linting option.
#### Description of Task to be completed?
Setup a simple react application with webpack.
```
git clone https://github.com/andela/ah-lobos-frontend.git
```
> navigate into the project directory
```
cd ah-lobos-frontend
```
> install all dependencies
```
npm install
```
> start application
```
npm start
```
> your browser should automatically open to display the text ``` Lobos Wolves``` as shown in the screenshot below
Screen Shot 2019-08-14 at 11 39 00

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166789823](https://www.pivotaltracker.com/story/show/166789823)
#### Screenshots (if appropriate)
Screen Shot 2019-08-13 at 21 41 06

#### Questions:
N/A

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>ah-lobos-frontend</title>
+</head>
+<body>
+    <div id='app'></div>
+</body>
+</html>
```
Comment (Dom58) [inline src/index.html]: New line issue

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ]
+}
```
Comment (Dom58) [inline .babelrc]: New line issue

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            80
+        ]
+    },
+    "settings": {
+        "react": {
+            "version": "detect"
+        }
+    },
+    "root": true
+}
```
Comment (Dom58) [inline .eslintrc]: New line issue

**Review by manzif — APPROVED** [review]

**Review by Emile-Nsengimana — APPROVED** [review]

**Review by j4l13n — APPROVED** [review]

**Review by Dom58 — APPROVED** [review]

**Review by kamyaD — COMMENTED** [review]
Great job. Remember to specify in the ignored files whether its a directory or a file. for example node_modules is a directory and should be marked by a "/".

**Comment by adafia** [comment]
> Great job. Remember to specify in the ignored files whether its a directory or a file. for example node_modules is a directory and should be marked by a "/".

Thank you very much @kamyaD, I'll fix it and push again.

**Comment by adafia** [comment]
> > Great job. Remember to specify in the ignored files whether its a directory or a file. for example node_modules is a directory and should be marked by a "/".
> 
> Thank you very much @kamyaD, I'll fix it and push again.

I have implemented your feedback, please review

**Review by kamyaD — COMMENTED** [review]
On your Description of tasks to be completed:
"The setup of a simple react application with webpack". Please remove "The" at the beginning of the description. It should read: "Setup a simple react application with webpack"

- Also put a full description of How to manually test the feature. Don't refer them to the README

**Comment by adafia** [comment]
> On your Description of tasks to be completed:
> "The setup of a simple react application with webpack". Please remove "The" at the beginning of the description. It should read: "Setup a simple react application with webpack"
> 
> * Also put a full description of How to manually test the feature. Don't refer them to the README

Thank you @kamyaD , I have implemented the above feedback, could you please check it again.

**Review by kamyaD — APPROVED** [review]
Great job. Thanks for acting quickly on feedback

**Pull request merged**